# A goal Position: X,Y,Z and Orientation
#x_msgs Point position

  int64 msg_id

  int64 pos_id

  int64 io_num

  float64 x
  float64 y
  float64 z

  float64 j1
  float64 j2
  float64 j3
  float64 j4
  float64 j5
  float64 j6



